README.md

```diff
@@ -21,6 +21,10 @@ Documentation: https://docs.joern.io/
 
 Specification: https://cpg.joern.io
 
+## News / Changelog
+We maintain a [dedicated page](NEWS.md) for news on API-breaking changes.
```

Comment:
would make sense to maintain a TOC for the changelog here, i.e. add one entry
`* v2.0.0 (June 2023) removal of Traversal` or so, that links to the section in the changelog

Reply:
I like that idea. Will do something like it.